extensions/nns/Cargo.toml

```diff
@@ -46,5 +50,4 @@ pre-release-replacements = [
 ]
 
 [package.metadata.dist]
-include = ["extension.json"]
-
+include = ["extension.json", "sns-cli", "ic-admin", "ic-nns-init"]
```

Comment:
cargo-dist's support to include directories is TBD: https://opensource.axo.dev/cargo-dist/book/config.html#include

I don't know if you mean: `put all bundled binaries in`
- `CACHE/extensions/bundled-binaries`
- or in `CACHE/extensions/EXT/bundled-binaries`

?

I'm fine with the second, but I don't like the first because it would prevent developers from installing multiple versions of the extensions